gap/PGMatrix.gd

```diff
@@ -0,0 +1,96 @@
+#!
+#!
+#! @Chapter Hyperbolic lattice symmetries
+
+#!
+#! @Section Point group matrices
+#! @SectionLabel PGMatrix
+#!
+#! Symmetry elements of triangle groups might act non-trivially on hyperbolic momenta. 
+#! As such, the degrees of freedom of observables and topological invariants, like the 
+#! first Chern numbers in momentum space, might be contstrained by the hyperbolic lattice 
+#! symmetries of underlying models. We can determine how hyperbolic momenta transform as 
+#! hyperbolic lattice symmetry transformations act on Abelian states. This can be extracted 
+#! by constructing point-group matrices.
+
+#! @Description
+#!   Construct point-group matrices for hyperbolic lattice <A>symmetries</A>, given as words in 
+#!   the triangle group <A>fulltgFp</A> ($\Delta$), a finitly presented group (see <Ref Sect="Section_TriangleGroups"/>), 
+#!   which is either a single symmetry or a list of symmetries for a unit cell <A>cell</A>, 
+#!   given as a `TGCell` object (see <Ref Sect="Section_TGCellGraph"/>).
```

Comment:
I'm a bit unsure about the arguments here. First, do you need a `TGCell` object or a `TGCellGraph` (you link to the second). Second, why do you need the `quotient` argument?